Dockerfile

```
@@ -1,43 +1,47 @@
-FROM python:3.12.6
+FROM python:3.12.6-slim as base
```

Nit to avoid linting errors docker will put out because of inconsistent reserved word casing.
```suggestion
FROM python:3.12.6-slim AS base
```